guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc

```diff
@@ -1,16 +1,13 @@
 [id='retrieving-metrics-in-the-web-ui_{context}']
 = Retrieving Metrics in the Web UI
 
-The Grafana web application provides a dashboard-style view, with default widgets displaying the values of metrics.
+You can view the Grafana dashboard, which provides default widgets displaying the values of metrics.
 You can add and remove metrics to suit your requirements.
-
-You can also select the time span shown for each widget.
-
-For more details on using Grafana, see the https://grafana.com/[Grafana Labs] website.
+You can also select the time span displayed for each widget.
 
 .Procedure
-. Open the following URL:
-http://_{foreman-example-com}_:3000
+* Open the following URL in a browser: http://_{foreman-example-com}_:3000
 
-.Example of the Grafana dashboard
-image::grafana_screenshot.png[PCP Redis Host Overview dashboard]
```

Comment:
We generally try to avoid screenshots. Can I have an ack from peers?

Reply:
Yeah.